### DISCUSSION TITLE
Gephi skipping edges

### DISCUSSION THREAD

**Discussion by DAAVLIB** [gephi/gephi#2818] [Q&A]
Hello there,
First time poster. I'm doing a citation analysis that has directed edges (author A cites author B) and undirected edges (author A co-authors with author B). These are all in the same edge file and when I upload it to Gephi (0.10.1), almost all of my undirected edges are skipped. I tried creating two separate files. When I upload the directed file first, I get the same result. When I upload the undirected file first, I get slightly more results (although not all) along with an error message. I tried the Don't Merge feature and the Sum feature and paradoxically, the Don't Merge feature seems to create higher weights for some authors than the Don't Merge feature. Anyway, it's a bit of a mess and I'm hoping to get some guidance. I thank you in advance. 

M

**Comment by mbastian** [comment]
Thanks for the question. What exactly are you trying to do? Merge all edges into one or do you want to have multiple edge type? For instance one type co-authorship and one type citation?

**Comment by DAAVLIB** [comment]
Hello Matthieu.

Thank you very much for getting back to me.

In a perfect world, it would be great to have multiple edge types to show whether the relationship is directed (citation) or undirected (co-authorship).

***@***.***

Right now, I have tried to “solve” the issue by having a file that creates a two-way directed relationship if there is a co-authorship. So both citations and co-authorships get absorbed in the same directed line. It’s not ideal and I’m hoping that I can show both types of relationships on the same graph.

Thanks for taking the time to look into this – I appreciate it.

Mario
